### DISCUSSION TITLE
Stuck on admin user creation.

### DISCUSSION THREAD

**Discussion by configuration-ninja** [Budibase/budibase#7709] [Help]
Hello there!
I am trying to setup Budibase on Kubernetes cluster in version v1.24.2-gke.1900 hosted on Google Cloud Platform. To deploy helm chart I am using Argocd. This is 100% default helm package deployment and after installation I cannot go past Admin creation screen.
On screen **Set budibase organization** I am moving forward, next on **Create an admin user** randomly I can or cannot create user. In other words, on the first try I go message that user has been created successfully, but I am stuck on **Create an admin user**, on second I got red popup with `Failed to create admin user` message.
I checked pods and looks like everything is running fine, no restart. Sadly I am really new to Budibase and I am not sure where to look for admin creation logs etc.

Hosted: GCP
Kubernetes: v1.24.2-gke.1900
Ingress used default
Helm chart version: 0.2.11 (latest available in helm chart repo)

**Comment by AiCaptain** [comment]
See https://github.com/Budibase/budibase/discussions/7699

Probably, you can see the status code 503 and error message "Unable to retrieve user list" in your browsers dev-tools/network , too.


Regards
André

**Comment by configuration-ninja** [comment]
@AiCaptain Thanks 🙇🏻🙇🏻🙇🏻  for suggestion and link to #7699 in fact I am getting 500 when requesting for `http://xxx/api/global/users/init`. I will try to override version to v1.2.52. Thank for sharing and quick replay!

**Comment by ht512512** [comment]
I am facing the same problem with my local docker-compose installation. Any suggestions? 
Thanks.

**Comment by krzwiatrzyk** [comment]
Same, unfortunately  nothing in the logs indicate what is an issue (Kubernetes version)
